bulbs/super_features/serializers.py

```diff
@@ -41,6 +41,10 @@ class BaseSuperFeatureSerializer(ContentSerializer):
     class Meta:
         model = BaseSuperFeature
 
+    def to_representation(self, obj):
```

Comment:
just adding this for documentation purposes, since this serializer isn't being used